.github/CODEOWNERS

```diff
@@ -587,6 +587,13 @@ x-pack/plugins/session_view @elastic/awp-platform
 x-pack/plugins/security_solution/public/common/components/sessions_viewer @elastic/awp-platform
 x-pack/plugins/security_solution/public/kubernetes @elastic/awp-platform
 
+## Security Solution sub teams - Protections Experience
+x-pack/plugins/threat_intelligence @elastic/protections-experience
```

Comment:
@maxcold do you have what you need to get this GitHub team added to the Kibana repo, for future PRs?

Reply:
yes, how do we have it added to the Kibana repo?

Reply:
asked in #kibana slack channel